### DISCUSSION TITLE
Welcome to AgeCalc Discussions!

### DISCUSSION THREAD

**Discussion by AthithKrishnaA** [AthithKrishnaA/AgeCalc#5] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AthithKrishnaA** [comment]
share ideas on improving this repo

**Comment by anirudhps1066** [comment]
Keep it up Bro..

**Comment by anirudhps1066** [comment, marked as answer]
We can Remove the calendar icon on the right most side
